cloud/docker_test.go

```diff
@@ -61,10 +61,14 @@ func (s *DockerSuite) SetupTest() {
 	s.hostOpts = HostOptions{
 		ParentID: "d",
 	}
-	s.parentHost.Insert()
-	s.distro.Insert()
+	s.NoError(s.parentHost.Insert())
+	s.NoError(s.distro.Insert())
 }
 
+func (s *DockerSuite) TearDownTest() {
+	s.NoError(db.ClearCollections(host.Collection))
```

Comment:
I think this function can take a list `db.ClearCollections(host.Collection, distro.Collection)`